crates/web-sys/webidls/enabled/Streams.webidl

```diff
@@ -71,10 +75,10 @@ interface mixin ReadableStreamGenericReader {
 
 [Exposed=*]
 interface ReadableStreamDefaultReader {
-  constructor(ReadableStream stream);
+  [Throws] constructor(ReadableStream stream);
 
   Promise<ReadableStreamReadResult> read();
-  undefined releaseLock();
+  [Throws] undefined releaseLock();
```

Comment:
I don't see any way in the spec that this can fail... looking at Mozilla's implementation, though, it can throw an internal error, which is probably why this is here.

Reply:
Addressed in new commits